tests/PHPStan/Analyser/data/callables.php

```diff
@@ -12,12 +12,17 @@ public function doFoo(): float
 		};
 		$foo = $this;
 		$arrayWithStaticMethod = ['Callables\\Foo', 'doBar'];
-		$stringWithStaticMethod = 'Callables\\Foo::doFoo';
+		$stringWithStaticMethod = 'Callables\\Foo::doBaz';
```

Comment:
Please do not modify existing tests. If they started failing and if you think that's for a good reason, always adjusts the assertions, not the test data. And if you want to test something new, always add new tests. Thanks.